### PR TITLE
Feat/eae 11 setup framework

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,56 +1,23 @@
...
-      </v-btn>
-    </v-app-bar>
-
-    <v-content>
-      <HelloWorld/>
-    </v-content>
+    <side-menu />
+    <v-main>
+      <router-view />
+      <map-component />
```
Comment (SiggyF) [inline src/App.vue:6]: Why is map-comonent not in a view?

File: src/css/markdown.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
+.markdown p {
```
Comment (SiggyF) [inline src/css/markdown.css:1]: Where did this style come from?
Should this use vuetify sass variables for consistency?
see https://vuetifyjs.com/en/customization/sass-variables/#variable-api

File: src/css/typography.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+
+.strong {
+  font-weight: bold;
+}
+
+.h1,
```
Comment (SiggyF) [inline src/css/typography.css:26]: See sass `$headings` variable

**Review by SiggyF — APPROVED** [review]
Minor comments

File: src/css/markdown.css (hunk 2 above)
Comment (CindyvdVries) [reply src/css/markdown.css:1]: this is the style for the imported markdown files - > vue-markdown

**Pull request merged**